lms/services/blackboard_api/_schemas.py

```diff
@@ -8,46 +8,35 @@
 from lms.validation._base import RequestsResponseSchema
 
 
-class _FileSchema(Schema):
-    """Shared base schema for Blackboard file dicts."""
-
-    class Meta:
-        unknown = EXCLUDE
-
-    id = fields.Str(required=True)
-    display_name = fields.Str(data_key="name", required=True)
-    updated_at = fields.Str(data_key="modified", required=True)
-    mime_type = fields.Str(data_key="mimeType")
+class BlackboardListFilesSchema(RequestsResponseSchema):
+    """Schema for Blackboard API /courses/{courseId}/resources responses."""
 
-    @post_load
-    def post_load(self, data, **_kwargs):  # pylint:disable=no-self-use
-        data["id"] = f"blackboard://content-resource/{data['id']}/"
-        return data
+    class FileSchema(Schema):
+        """Schema for individual Blackboard file dicts."""
 
+        class Meta:
+            unknown = EXCLUDE
 
-class BlackboardListFilesSchema(RequestsResponseSchema):
-    """Schema for Blackboard API /courses/{courseId}/resources responses."""
+        id = fields.Str(required=True)
+        name = fields.Str(required=True)
+        modified = fields.Str(required=True)
+        mimeType = fields.Str()
```

Comment:
The diff is a bit confusing because I also inlined `_FileSchema` into `BlackboardListFilesSchema` at the same time. The `FileSchema` class is actually exactly the same as before but without the `data_key`'s and the `@post_load`